### DISCUSSION TITLE
Add sync capabilities of files outside of HA backups

### DISCUSSION THREAD

**Discussion by lavinir** [lavinir/hassio-onedrive-backup#15] [Polls]
I got a request to add the ability to allow file syncing to OneDrive from Home Assistant outside of the backups.
The use case was that for very large files this takes up a lot of unnecessary space in each backup (i.e. media)

Trying to gauge how much this is something people will use.

**Comment by felipecrs** [comment]
This would be really nice. For example, I'd set the automatic backup to exclude media folder (which contains all my Frigate recordings), but would enable this feature to sync `/media/frigate/clips` which are only clips of detected objects.

**Comment by lavinir** [comment]
1 way File Syncs are now available in the 1.4 Release.
